packages/client/src/runtime/utils/common.ts

```diff
@@ -84,6 +85,7 @@ export const JSTypeToGraphQLType = {
   string: 'String',
   boolean: 'Boolean',
   object: 'Json',
+  symbol: 'Symbol',
```

Comment:
Is this a valid type and can this be ever reached? Or does it have another purpose?

Reply:
It's reachable when a symbol that is not a valid enum value is passed (maybe even just some random JavaScript symbol). We need something here so that `getGraphQLType` doesn't return `undefined`. The exact name of the type isn't super important, it will basically only be used in the validation error message.

Reply:
ah no, it's actually also used in one check in `hasCorrectScalarType`

Reply:
So, now it's not really used in this PR anymore, but it's still nice to have for better validation errors if an unexpected JS symbol is passed to whatever field.

Should I keep it here, or move it to a separate PR (with a test for that optimally)?

Reply:
I think it's ok to keep indeed.